sys/include/net/dhcpv6/client.h

```diff
@@ -131,6 +138,17 @@ void dhcpv6_client_start(void);
 void dhcpv6_client_req_ia_pd(unsigned netif, unsigned pfx_len);
 /** @} */
 
+/**
+ * @brief   Configures the client to request non-temporary addresses for a network
+ *          interface from a server
+ * @note    For WPAN meshes a DHCPv6 relay (which is not implemented in RIOT yet) is
+ *          required
```

Comment:
```suggestion
 * @note    For multi-hop WPAN meshes a DHCPv6 relay (which is not implemented in
 *          RIOT yet) is required, as DHCPv6 only acts in link scope.
```

should make clearer what is meant here.